### ISSUE TITLE
PDF export on Android not working

### ISSUE THREAD

**Issue by Gustl22** [Oberhauser-Dev/wrestling_scoreboard#60]

**Comment by Zomtir** [comment]
*Not* fixed by adding those external storage permissions in the AppManifest:
```
    <!-- Storage permissions for Android <=9 -->
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
    <!-- Storage permissions for Android >=10 -->
    <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" />
```

**Comment by Zomtir** [comment]
Fix available in 6023958d65199c5d1e39f70f6928e8e087b2a02d

Issue was that `file_selector.getSaveLocation()` is not available on Android. However `file_selector.getDirectoryPath()` is implemented...

Necessity of storage permissions not tested.
